package org.jpvm.excptions.pyobjs;public class PyAttrubuteErrorObject {
}
